auth_admin_passkey/models/res_users.py

```diff
@@ -53,7 +53,9 @@ def _prepare_email_passkey(self, login_user):
             "login": login_user.login,
             "login_date": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
         }
-        return subject, "<pre>{}</pre>".format(body)
+        return subject, f"<pre>{body}</pre>"`
+
+looks lighter.
```

Comment:
sorry. something has been wrong in the github proposal !

Reply:
@legalsylvain I fixed this issue. Now I have the following issue:
AttributeError: 'dict' object has no attribute 'encode' on line:
password = hashlib.sha512(password.encode()).hexdigest()

Reply:
it looks quite unrelated, don't you think ?

Reply:
@legalsylvain Yes your solution fixed the previous problem. It's not related to this one.